aries-site/src/examples/components/notifications/StateVsStatusDiagram.js

```diff
@@ -4,7 +4,7 @@ import { ToastPreview } from '../../cardPreviews';
 
 const color = 'text-strong';
 const anchor = 'vertical';
-const thickness = 'xxxsmall';
+const thickness = 'hair';
```

Comment:
opportunistic propTypes validation error --> 'hair' is valid value, 'xxxsmall' is not